contrib/win-installer/build.ps1

```diff
@@ -99,7 +99,7 @@ if ($args.Count -lt 1 -or $args[0].Length -lt 1) {
 }
 
 # Pre-set to standard locations in-case build env does not refresh paths
-$Env:Path="$Env:Path;C:\Program Files (x86)\WiX Toolset v3.11\bin;C:\ProgramData\chocolatey\lib\mingw\tools\install\mingw64\bin;;C:\Program Files\Go\bin"
+$Env:Path="$Env:Path;C:\Program Files (x86)\WiX Toolset v3.14\bin;C:\ProgramData\chocolatey\lib\mingw\tools\install\mingw64\bin;;C:\Program Files\Go\bin"
```

Comment:
Windows Experts PTAL. I don't know what WiX is, nor why it's needed, nor if 3.14 is compatible with 3.11.

Reply:
@n1hility @baude @ashley-cui @mheon PTAL

Reply:
It's a windows installer toolkit. 3.14 is a security update, so we want to grab it.

Reply:
+1